onnxruntime/core/providers/shared_library/provider_interfaces.h

```diff
@@ -1354,6 +1354,11 @@ struct ProviderHost {
   virtual std::unique_ptr<ModelMetadefIdGenerator> ModelMetadefIdGenerator__construct() = 0;
   virtual void ModelMetadefIdGenerator__operator_delete(ModelMetadefIdGenerator* p) = 0;
   virtual int ModelMetadefIdGenerator__GenerateId(const ModelMetadefIdGenerator* p, const GraphViewer& graph_viewer, HashValue& model_hash) = 0;
+
+#ifdef _WIN32
+  virtual std::string ToUTF8String(std::wstring_view s) = 0;
```

Comment:
Is it an ABI change? Adding new functions to the EP interface.

Reply:
In the previous PR, #25338, you asked us to use wide strings for Windows paths and string views for literals. The EP interface is missing methods for the wide string view type.

Reply:
It's done. I have removed those abstract methods from the EP interface.